### ISSUE TITLE
Multilabel Classification

### ISSUE THREAD

**Issue by foodiehack** [creme-ml/creme#347]
Is it possible to do multilabel classification with creme-ml. Can't find anything in the docs

**Comment by MaxHalford** [comment]
Hey there @foodiehack. Yes, you can do multi-label classification with `creme`. Take a look at [`ClassifierChain`](https://creme-ml.github.io/api-reference/multioutput/ClassifierChain/).

**Comment by foodiehack** [comment]
@MaxHalford So I am initially training a model with 5 labels. On a weekly basis, 1k data appears and it is possible that new label comes too. Is there a way to retrain the model with that new label too without training data from scratch?

**Comment by MaxHalford** [comment]
> Is there a way to retrain the model with that new label too without training data from scratch?

Yep, that's the intention. At least, every algorithm in `creme` is designed from the ground up to support new features, classes, and outputs to be added on-the-fly. However, this isn't something that we have tested a 100% because of lack of resources. 

In short: yes, it should work. If it doesn't, then please tell us and we'll get it fixed.

**Comment by foodiehack** [comment]
@MaxHalford Thanks for the prompt reply. I will try it out. As I can see in the example Classfier Chain is used with linear model LogisticRegression. 
Can I somehow use LinearSVC or SGD(loss='modified_huber'') along with Classifier Chain or any other classification algo.

**Comment by MaxHalford** [comment]
I assume that you're referring to models from scikit-learn's `linear_model` module? If so I'm not sure what you're asking. You can use any `creme` classifier as an input to the classifier chain.
